P/PETSc/build_tarballs.jl

```diff
@@ -280,7 +280,7 @@ dependencies = [
     Dependency("OpenBLAS32_jll"; platforms=filter(!Sys.isapple, platforms)),
     Dependency("CompilerSupportLibraries_jll"),
     Dependency("SuperLU_DIST_jll"; compat=SUPERLUDIST_COMPAT_VERSION),
-    Dependency("SuiteSparse_jll"; compat=SUITESPARSE_COMPAT_VERSION),
+    Dependency("SuiteSparse_jll", compat=SUITESPARSE_COMPAT_VERSION),
```

Comment:
Why this change?  In my opinion `;` is better when separating positional arguments from keyword ones.